kv/kvserver: TestSplitWithExternalFilesFastStats failed

kv/kvserver.TestSplitWithExternalFilesFastStats [failed](https://mesolite.cluster.engflow.com/invocations/default/98e9de93-e9b9-4442-a4ed-8626f233e5a7?testReportRun=1&testReportShard=21&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZzZXJ2ZXI6a3ZzZXJ2ZXJfdGVzdA==) on release-24.3.25-rc @ [eecd7c2c316ed213daab1c664e0d854fc937a746](https://github.com/cockroachdb/cockroach/commits/eecd7c2c316ed213daab1c664e0d854fc937a746):


```
=== RUN   TestSplitWithExternalFilesFastStats
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestSplitWithExternalFilesFastStats2217616769
    test_log_scope.go:76: use -show-logs to present logs inline
    client_split_test.go:4534: -- test log scope end --
test logs left over in: outputs.zip/logTestSplitWithExternalFilesFastStats2217616769
--- FAIL: TestSplitWithExternalFilesFastStats (5.37s)
=== RUN   TestSplitWithExternalFilesFastStats/kv.range_split.estimated_stats_for_external_bytes.enabled=true
    client_split_test.go:4507: 
        	Error Trace:	pkg/kv/kvserver/client_split_test.go:4507
        	Error:      	"0" is not greater than "0"
        	Test:       	TestSplitWithExternalFilesFastStats/kv.range_split.estimated_stats_for_external_bytes.enabled=true
        	Messages:   	lhs expected to have estimated stats
    --- FAIL: TestSplitWithExternalFilesFastStats/kv.range_split.estimated_stats_for_external_bytes.enabled=true (2.57s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=21</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestSplitWithExternalFilesFastStats.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-58470

--- a/pkg/kv/kvserver/client_split_test.go
+++ b/pkg/kv/kvserver/client_split_test.go
@@ -4391,6 +4391,7 @@ func TestSplitWithExternalFilesFastStats(t *testing.T) {
 					Store: &kvserver.StoreTestingKnobs{
 						DisableMergeQueue:              true,
 						DisableSplitQueue:              true,
+						DisableGCQueue:                 true,
 						DisableCanAckBeforeApplication: true,
 					},
 				},

